tests/conftest.py

```diff
@@ -49,6 +50,19 @@ def fx_tmpdir(tmpdir):
 
 
 @yield_fixture
+def fx_change_cwd_to_python_dir():
```

Comment:
I think that "change current working directory to python dir" has same meaning "cd python dir". :3